lighthouse-core/gather/gatherers/dobetterweb/all-event-listeners.js

```diff
@@ -118,18 +118,16 @@ class EventListeners extends Gatherer {
     })).then(nestedListeners => [].concat(...nestedListeners));
   }
 
-  beforePass(options) {
```

Comment:
this change fixes the 2nd half of https://github.com/GoogleChrome/lighthouse/issues/2163

and doing it this way means we don't need pavel's CL 

\o/

patrick, you should followup with pavel separately if you'd rather migrate to violations vs this approach. but this can be post-IO